pages/message-sent/index.tsx

```diff
@@ -3,7 +3,9 @@ import { Helmet } from "react-helmet";
 import React from "react";
 import LayoutContainer from "../../component/LayoutContainer";
```

Comment:
please change relative import to alias import
e.g
import { BannerPage } from "component/Page/BannerPage"